tests/test_reveal.bats

```diff
@@ -293,3 +293,25 @@ function teardown {
   # clean up
   rm -rf subdir
 }
+
+@test "run 'reveal' for all files from subdir" {
+  local password
+  password=$(test_user_password "$TEST_DEFAULT_USER")
+
+  mkdir subdir
+  echo "content2" > subdir/new_filename.txt
+
+  ( # start subshell for subdir tests
+    cd subdir
+    run git secret add new_filename.txt
+    [ "$status" -eq 0 ]
+    run git secret hide
+    [ "$status" -eq 0 ]
+  
+    run git secret reveal -d "$TEST_GPG_HOMEDIR" -p "$password"
```

Comment:
Let's also test that `reveal new_filename.txt` will also work!

Reply:
The test case already exists: https://github.com/ochanism/git-secret/blob/4aa2229904852e2d4b76515801a1b2e62775d3d2/tests/test_reveal.bats#L275

I copied and modified it. Do you want to merge them?

Reply:
Ok then, let's leave it as-is.